create-neon/src/bin/create-neon.ts

```diff
@@ -29,31 +30,60 @@ function inferVersions(): Versions {
   };
 }
 
+function deleteNeonDir(dir:string): Promise<void> {
+  return fs.rm(dir, { force: true, recursive: true });
+}
+
 async function main(name: string) {
+  let workingDirPath: string = process.cwd()
+  let tmpDirPath: string = os.tmpdir()
+  let tmpFolderName:string|undefined;
+
   try {
-    await fs.mkdir(name);
+    //check if folder or file with same name exists
+    await fs.mkdir(name)
+    await fs.rmdir(name)
```

Comment:
I like the `mkdir` and `rmdir` because it's a pretty lightweight way to check _both_ that it doesn't exist *and* that the user has write permissions.

Can you add a comment explaining this benefit?